js/web/lib/wasm/jsep/init.ts

```diff
@@ -186,10 +186,10 @@ export const init = async(module: OrtWasmModule, env: Env, gpuAdapter: GPUAdapte
           },
 
       // jsepCreateKernel
-      (name: string, kernel: number, attribute: unknown) => backend.createKernel(
-          name, kernel, attribute,
-          env.debug || backend.queryType !== QueryType.none ? module.UTF8ToString(module._JsepGetNodeName(kernel)) :
-                                                              `${kernel}`),
+      (opType: string, kernelId: number, attribute: unknown) => backend.createKernel(
+          opType, kernelId, attribute,
+          env.debug || backend.queryType !== QueryType.none ? module.UTF8ToString(module._JsepGetNodeName(kernelId)) :
+                                                              `${opType}`),
```

Comment:
What's the cost of module._JsepGetNodeName()? Can we call it unconditionally?

Reply:
cc @fs-eire

Reply:
I didn't measure that. We can do that unconditionally for now and if future profiling data shows it's heavy we can remove it then.